### PR TITLE
fix for AR-396 menu close button

### PR DESCRIPTION
**Pull request by Zoharz**

### CODE REVIEW

File: src/components/Layout/Layout.js [hunk 1]
```diff
@@ -132,7 +137,8 @@ class Layout extends Component {
                 }
               }}
             >
-              <Icon name="sidebar" />
+              {/* <Icon name="sidebar" /> */}
```
Comment (edoshor) [inline src/components/Layout/Layout.js]: Please remove commented out code

File: src/components/Layout/Layout.js [hunk 2]
```diff
@@ -186,7 +192,8 @@ class Layout extends Component {
...
```
Comment (edoshor) [inline src/components/Layout/Layout.js]: here as well

File: src/components/Layout/Layout.js (hunk 1 above)
Comment (Zoharz) [reply src/components/Layout/Layout.js]: done

File: src/components/Layout/Layout.js (hunk 2 above)
Comment (Zoharz) [reply src/components/Layout/Layout.js]: done

**Pull request merged**